tests/vmcheck/test-basic.sh

```diff
@@ -59,3 +59,17 @@ echo "ok status doesn't require root"
 # Also check that we can do status as non-root non-active
 vm_cmd runuser -u bin rpm-ostree status
 echo "ok status doesn't require active PAM session"
+
+# Add metadata string containing EnfOfLife attribtue
+META_ENDOFLIFE_MESSAGE="this_is_a_test"
+commit=$(vm_cmd ostree commit -b vmcheck \
+            --tree=ref=vmcheck --add-metadata-string=ostree.endoflife=$META_ENDOFLIFE_MESSAGE)
```

Comment:
Then this would need to be `--add-metadata-string=ostree.endoflife="${META_ENDOFLIFE_MESSAGE}"`.